### PR TITLE
Mobile : Disabled the header items when no note is selected

### LINKED ISSUE #2987: Mobile : Header items remain enabled even when no note is selected
## Environment

Joplin version: 1.0.325
Platform: Android
OS specifics: No

## Steps to reproduce

1. Select one or more notes 
2. Deselect all by toggling the **select all** button
3. The **Move to notebook**, **delete** and **duplicate** items on the header remains enabled

Although they are enabled, but still they can't perform any functionality.

## Describe what you expected to happen
**Move to notebook**, **delete** and **duplicate** items should disable when no note is selected & enable only when one or more notes are selected.

### PR DESCRIPTION
**Pull request by Rahulm2310**
**Before:**

**After:** 
joplinerrorsolve

fixes #2987

### CODE REVIEW

**Comment by Rahulm2310** [comment]
@PackElend sir please label me

File: ReactNativeClient/lib/components/Dropdown.js [hunk 1]
```diff
@@ -112,6 +112,7 @@ class Dropdown extends React.Component {
 				<TouchableOpacity
 					style={headerWrapperStyle}
 					ref={ref => (this.headerRef_ = ref)}
+					disabled={this.props.dropdownDisabled}
```
Comment (laurent22) [inline ReactNativeClient/lib/components/Dropdown.js:115]: Please name it "props.disabled"

Comment (Rahulm2310) [reply ReactNativeClient/lib/components/Dropdown.js:115]: Okay sir, I will do it.

**Comment by Rahulm2310** [comment]
@laurent22 I have made the required changes

**Comment by laurent22** [comment]
Looks good, thanks @Rahulm2310

**Comment by Rahulm2310** [comment]
Thanks! @laurent22

**Pull request merged**